package.json

```diff
@@ -8,13 +8,14 @@
     "build": "babel . -d build --copy-files --ignore node_modules,postman,build",
     "init-db": "babel-node src/db/init.js",
     "lint": "eslint index.js src/",
-    "test": "mocha --reporter spec src/**/*.test.js --require babel-register"
+    "test": "find ./src -name '*tests.js' | xargs mocha --require babel-register"
```

Comment:
i think should be problems with this script for windows users, @tyndria or @asntr check it please.

PS: I don't find how to run mocha with glob pattern, it processed only one test file, mb somebody find solving?